File: CompanyRatingProject/DTO/ModelViewsObjects/RatingDTO.cs
namespace DTO.ModelViewsObjects;

public class RatingDto
{
    public string Id { get; set; }
    public int Grade1 { get; set; }
    public int Grade2 { get; set; }
    public int Grade3 { get; set; }
    public int Grade4 { get; set; }
    public int Grade5 { get; set; }
    public double Total { get; set; }
    public string Text { get; set; }
    public DateTime DateTime { get; set; }
    public CompanyDto Company { get; set; }
    public string CompanyId { get; set; }
    public UserDto User { get; set; }
    public string UserId { get; set; }
}

File: CompanyRatingProject/CompanyRatingProject/ClientApp/src/app/DTO/RatingDTO.ts
import {DatePipe} from "@angular/common";
import {UserDTO} from "./UserDto";

export class RatingDTO{
    constructor(
        public id?:string,
        public grade1?:number,
        public grade2?:number,
        public grade3?:number,
        public grade4?:number,
        public grade5?:number,
        public total?:number,
        public text?:string,
        public dateTime?:Date,
        public companyId?:string,
        public userId?:string,
        public user?:UserDTO
    ) {}
}